mod_test/controllers.py

```diff
@@ -233,6 +234,23 @@ def by_commit(commit_hash):
     return get_data_for_test(test, 'commit {commit}'.format(commit=commit_hash))
 
 
+@mod_test.route('/master/<platform>')
+@template_renderer('test/by_id.html')
+def latest_commit_info(platform):
+    try:
+        platform = TestPlatform.from_string(platform)
+    except ValueError:
+        return 'ERROR'
```

Comment:
Returning a simple 'ERROR' is not that user-friendly. I'd like to see a nicer error. Check on how to raise a 404.